src/chain.cpp

```diff
@@ -14,6 +13,9 @@ using namespace std;
  * CChain implementation
  */
 
+const CFieldElement CBlockIndex::defaultScCumTreeHash =
```

Comment:
Introduce a utility method to retrieve from zendoo_mc_cryptolib a constant phantom hash field element and use it everywhere it is needed a constant value whose preimage has to be unknown